Lib/importlib/_bootstrap_external.py

```diff
@@ -1638,15 +1645,21 @@ def find_spec(self, fullname, target=None):
                 is_namespace = _path_isdir(base_path)
         # Check for a file w/ a proper suffix exists.
         for suffix, loader_class in self._loaders:
+            # XXX: Why is ValueError caught here?
```

Comment:
This change worries me a little because I don't understand why the original code catches `ValueError` and then bails out.

Removing this like I did here does not result in test failures.

Reply:
If `git blame` isn't helpful then it can probably be removed.